src/components/select-user/index.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import truncate from 'lodash/truncate';
+import { CloseIcon } from '../icons';
+
+
+const Wrapper = styled.div`
+    display: flex;
+    align-items: center;
+    min-width: 350px;
+    margin-bottom: 16px;
+`;
+
+const UserImage = styled.div`
+    flex: 0;
+    text-align: left;
+`;
+
+const UserPhoto = styled.figure`
+    width: 32px;
+    height: 32px;
+    border-radius: 100%;
+    overflow: hidden;
+    margin: 0;
+
+    img {
+        max-width: 100%;
+    }
+`;
+
+const UserName = styled.div`
+    flex: 1 1 30%;
+    padding-top: 7px;
+    padding-left: 16px;
+    padding-bottom: 6px;
```

Comment:
Two notes about it:

### too close values
In cases were you have too close values as `7px` and `6px`, I prefer to use `6px` to keep the consistency.

In this case:

```suggestion
    padding: 6px 16px;
```

### shorthand
When we have more than one value for the same property, is recomended to use the [shorthand](https://developer.mozilla.org/pt-BR/docs/Web/CSS/padding) version. 

```js
// padding: top right left bottom; (structure of the values)
// padding: 6px (top and bottom) 16px (right and left)
```

Reply:
Thanks for the tip!